src/functions/meme/main.py

```diff
@@ -0,0 +1,54 @@
+import json
+import os
+from typing import Optional, Tuple
+
+import functions_framework
+import requests
+from flask.wrappers import Request
+
+MEME_FILE = "memes.json"
+
+NO_EMOJI = "https://cdn.discordapp.com/emojis/823403768448155648.webp"
+USAGE_GUIDE = "Usage: !meme <Meme name> <Top text> <Bottom text (Optional)>"
+UNSUPORTED_MEME = "Meme not in supported list."
+
+
+@functions_framework.http
+def meme(request: Request) -> Tuple[str, int]:
+    """Return url to generated meme."""
+
+    request_json: Optional[dict] = request.get_json(silent=True)
+
+    with open(MEME_FILE, "r", encoding="utf-8") as meme_file:
+        memes: dict = json.load(meme_file)
```

Comment:
Why not make `memes.json` a python dict file so that we don't need to open files ?

Reply:
Using a JSON seems like better practice than using a hard coded dictionary. Especially because it probably is going to be modified often.

Reply:
JSON or python CONSTANT is the same, we just use python import and not file read